packages/app-project/docker-compose.yml

```diff
@@ -34,3 +35,6 @@ services:
       - ./pages:/usr/src/packages/app-project/pages
       - ./src:/usr/src/packages/app-project/src
       - ./stores:/usr/src/packages/app-project/stores
+      - node_modules:/usr/src/node_modules
+volumes:
+  node_modules:
```

Comment:
Shared volumes can be convenient for local development. You can inspect them in the Docker Desktop app. This one should contain all the installed Node packages.

<img width="800" alt="Screenshot of the front-end-monorepo_node_modules volume in Docker Desktop, showing a listing of the volume contents." src="https://github.com/user-attachments/assets/79617087-352e-49f4-85d4-c165ad34b6ee">

Reply:
I added two `node_modules` volumes, so you can compare production vs. development installs in Docker Desktop.